src/common/Input.js

```diff
@@ -24,6 +27,11 @@ const InputText = (props: Props) => (
       required={props.required}
     />
     {props.icon && <div className="inputIcon">{props.icon}</div>}
+    {props.withCancel && (
```

Comment:
`withCancel` isn't necessarily needed, it could be also done in a way that you check whether `resetInput` was passed as prop